tests/README.md

```diff
@@ -65,11 +65,6 @@ $ atoum -bf tests/bootstrap.php -f tests/functional/Html.php -m tests\units\Html
 ```
 In `tests\units\Html::testConvDateTime`, you may need to double the backslashes (depending on the shell you use);
 
-If you want to run the API tests suite, you need to run a development server:
-
-```bash
-php -S localhost:8088 -t public tests/router.php &>/dev/null &
```

Comment:
Maybe the `phpunit/router.php` file can still be used to execute web tests.

Reply:
You need to run it on the correct port, which will already be used by the webserver for most people in a dev env.
Given how almost no one run these, it probably isn't worth it to spend too much time on it.